src/app/modules/group/pages/current-user/current-user.component.html

```diff
@@ -1,5 +1,18 @@
 <ng-container *ngIf="currentUser$ | async as currentUser">
   <div class="bg-white">
+    <div class="modify-section">
+      <div class="flex-1" i18n>
+        The following information is what what receive from France-IOI login platform.
+      </div>
+      <button
+        pButton
+        type="button"
+        class="modify-button alg-button p-button-rounded p-button-info"
+        i18n-label label="Modify"
+        icon="pi pi-pencil"
+        (click)="onModify()"
+      ></button>
```

Comment:
The button style and icon is unexpected:
![image](https://user-images.githubusercontent.com/16257159/165259617-dd74b29e-fef4-4658-8513-429a58956277.png)
versus
![image](https://user-images.githubusercontent.com/16257159/165259519-92dd3bb7-9b81-4d0e-80a7-8083bf13f973.png)